R/utils-side.R

```diff
@@ -122,22 +132,17 @@ locked_sides <- function(sides){
 
 #' set the log value on a side
 #' 
-#' @param view a single named view
 #' @param sides the sides for the gsplot object (see \code{\link{sides}})
-#' @return a modified \code{sides} list
+#' @return a modified \code{side}
 #' @keywords internal
-set_side_log <- function(view, sides){
-  stopifnot(length(view) == 1)
-  view.name <- names(view)
-  log <- summarize_args(view, c('log'), ignore=c('gs.config'), na.value = "")[[view.name]]
+set_side_log <- function(args, side, side.num){
```

Comment:
yeah, the number is lost with the side because it is only in the name in the list containing sides.